lang/cgilua/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cgilua
+PKG_VERSION:=5.1.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
You'll run into file names clashing if they having nothing more than v<version>.tar.gz as file name.